deploy/manifests/base/storetheindex/indexer-config.yaml

```diff
@@ -44,6 +44,7 @@ data:
         "Dir": "/data/datastore"
       },
       "Discovery": {
+        "FilterIPs": true,
```

Comment:
having the key present before the deploy of the version that recognizes it won't cause problems, right?

Reply:
Right, it gets ignored.